Phase03/ProgramController.cs

```diff
@@ -0,0 +1,40 @@
+using Newtonsoft.Json;
+namespace Phase03;
+public class ProgramController
+{
+    private static List<StudentData> students;
+    private static List<StudentGrade> grades;
+    public void Run()
+    {
+        InitializeLists();
+        FindAverageScores();
+        PrintResult(GetSortedAverages());
+    }
+    
+    private static void InitializeLists()
```

Comment:
Why are all the methods of this class (except for run) static?
Not only is the use of the static keyword is an anti-pattern but these methods have no need to be static because they are accessed by an instance of the class.